hashintel/hash
# Remove developer information from the homepage

## 🌟 What is the purpose of this PR?




This PR addresses a `@todo` to remove developer information from the homepage, and adds a redirect to the `/login` page.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203312852763953/1203451531168809/f) _(internal)_

## 📜 Does this require a change to the docs?



No.

## 🛡 What tests cover this?



None.

## ❓ How to test this?



1. Run the developer environment using `yarn dev`
2. Go to `localhost:3000` and ensure you are redirected to `localhost:3000/login`
3. Log in and ensure you are redirected to the workspace
4. Go to `localhost:3000` and ensure you are redirected to the workspace

## 📹 Demo



https://user-images.githubusercontent.com/42802102/204553968-1b4f4d19-61dd-420c-b525-c34c77b7e0b4.mov
